code/modules/genetics/recipes/cloning_recipes.dm

```diff
@@ -210,5 +210,5 @@
 	required_on_clone_types = list(/mob/living/simple_animal/hostile/nightmare)
 	required_mutations = list(MUTATION_EMP)
 
-/datum/genetics/mutation_recipe/combination/croaker_lord/get_result()
+/datum/genetics/mutation_recipe/combination/nightmare/get_result()
 	return new /datum/genetics/mutation/copy_mob(/mob/living/simple_animal/hostile/nightmare, "nightmare stalker")
```

Comment:
Doesn't that mean that cloning a nightmare stalker with the EMP mutation will still give the same nightmare stalker, or will it just give it the mutation?

Reply:
Honestly that's a pretty good point idk why I have it written that way